plasmapy/formulary/collisions/tests/test_frequencies.py

```diff
@@ -708,7 +753,7 @@ def test_fail1(self) -> None:
         """
         fail1 = self.True1 * (1 + 1e-15)
         with pytest.warns(PhysicsWarning, match="strong coupling effects"):
-            methodVal = collision_frequency(
+            methodVal: u.Quantity = collision_frequency(
```

Comment:
Is this type hinting necessary? It looks like collision_frequency has its own type hint that inidicates that it returns a Quantity. 

```python
def collision_frequency(
    T: u.Quantity[u.K],
    n: u.Quantity[u.m**-3],
    species,
    z_mean: float = np.nan,
    V: u.Quantity[u.m / u.s] = np.nan * u.m / u.s,
    method="classical",
) -> u.Quantity[u.Hz]:
```

Reply:
One possibility is that this was an automated refactoring by an IDE.  🤔  I've infrequently had unintentional changes from IDEs sneak in to pull requests before, like removing imports.  In any case, thanks for removing it!